SpatialGDK/Source/SpatialGDK/Public/Schema/RPCPayload.h

```diff
@@ -2,6 +2,7 @@
 
 #pragma once
 
+#include "Interop/Connection/SpatialGDKSpanId.h"
```

Comment:
Nit: doesn't seem to be used here

Reply:
It looks like there's quite a few places that include `RPCPayload.h` and use `SpatialGDKSpanId` but don't explicitly include `SpatialGDKSpanId.h`. Was the intention to transitively include it by adding it here?

Reply:
Nope! This was leftover, will fix up the other places